### PR TITLE
Add debris density

### PR DESCRIPTION
**Pull request by Baezon**
Just like ship density, allows for a ship class specific multiplier for debris mass and MOI, including subsystem debris-specific density.

Address the last point of #5278

### CODE REVIEW

**Review by wookieejedi — APPROVED** [review]
Nice and straightforward

File: code/debris/debris.cpp [hunk 1]
```diff
@@ -375,7 +375,7 @@ MONITOR(NumHullDebris)
  * @param hull_flag		Whether this debris is a chunk of a ship's hull (as opposed to shrapnel)
  * @param exp_force		Explosion force, used to assign velocity to pieces. 1.0f assigns velocity like before. 2.0f assigns twice as much to non-inherited part of velocity
  */
-object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d *pos, vec3d *exp_center, bool hull_flag, float exp_force)
+object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d *pos, vec3d *exp_center, bool hull_flag, float exp_force, ship_subsys* source_subsys)
```
Comment (JohnAFernandez) [inline code/debris/debris.cpp:378]: Could you add this new parameter to the doxy documentation above it?

**Review by JohnAFernandez — CHANGES_REQUESTED** [review]
Just one thing

File: code/debris/debris.cpp (hunk 1 above)
Comment (Baezon) [reply code/debris/debris.cpp:378]: whoops, ty

**Review by JohnAFernandez — APPROVED** [review]

**Pull request merged**